.mega-linter.yml

```diff
@@ -1,16 +1,10 @@
 ENABLE_LINTERS:
-  - PYTHON_PYLINT
```

Comment:
pre-commit job already runs whatever we have configured in our repo so those steps can be removed from the megalinter